include/nbl/builtin/hlsl/spirv_intrinsics/raytracing.hlsl

```diff
@@ -13,33 +13,47 @@ namespace hlsl
 namespace spirv
 {
 
+using RayQueryKHR = vk::SpirvOpaqueType<spv::OpTypeRayQueryKHR>;
+using AccelerationStructureKHR = vk::SpirvOpaqueType<spv::OpTypeAccelerationStructureKHR>;
```

Comment:
btw you may want to add `OpConvertUToAccelerationStructure` intrinsic, then one can use BDA instead of descriptor set if they so choose